### PR TITLE
support php v8

### PR DESCRIPTION
**Pull request by huzaif-plivo**

### CODE REVIEW

File: src/Plivo/Exceptions/PlivoResponseException.php [hunk 1]
```diff
@@ -29,7 +29,7 @@ class PlivoResponseException extends PlivoRestException
      * @param array $responseData
      * @param integer $statusCode
      */
-    public function __construct($message = "", $code = 0, Throwable $previous = null, $responseData, $statusCode)
+    public function __construct(?string $message, ?int $code, $responseData, $statusCode)
```
Comment (narayana-plivo) [inline src/Plivo/Exceptions/PlivoResponseException.php:32]: nullable works in php 8
Few question:
1) previously status code is zero , here is nullable
2)previous param is missing

Comment (huzaif-plivo) [reply src/Plivo/Exceptions/PlivoResponseException.php:32]: @narayana-plivo
1. There is no user-defined exception code, however, we have `$statusCode` param defined to fetch the HTTP status code.
2. The `$previous` param wasn't used anywhere in the code.
Should this be reverted?

**Review by narayana-plivo — APPROVED** [review]
pushed the changes and tested on php 8 and php 7.4.8 . 

**Note: On php 7.1 onwards this Nullable types supported.**

File: src/Plivo/Exceptions/PlivoResponseException.php (hunk 1 above)
Comment (narayana-plivo) [reply src/Plivo/Exceptions/PlivoResponseException.php]: 2. there are 24 reference for PlivoResponseException. Please careful on changing functional params. I have pushed the change and mentioned version too.

Comment (huzaif-plivo) [reply src/Plivo/Exceptions/PlivoResponseException.php]: Sure

**Pull request merged**